modules/generate-diagnostic-bundle-using-roxctl-cli.adoc

```diff
@@ -1,19 +1,64 @@
 // Module included in the following assemblies:
 //
 // * configuration/generate-diagnostic-bundle.adoc
-:_module-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="generate-diagnostic-bundle-using-roxctl-cli_{context}"]
 = Generating a diagnostic bundle by using the roxctl CLI
 
 You can generate a diagnostic bundle by using the `roxctl` CLI.
 
+.Prerequisites
+* To generate a diagnostic bundle, you need `read` permission for the `Administration` resource. This is required for versions of the `DebugLogs` resource older than version 3.73.0.
+
+* You have configured the `ROX_PASSWORD` and the `ROX_CENTRAL_ADDRESS` environment variables.
+
+** Run the following command to set the `ROX_PASSWORD` and the `ROX_CENTRAL_ADDRESS` environment variables:
++
+[source,terminal,subs="+quotes"]
+----
+$ export ROX_PASSWORD=_<rox_password>_ && export ROX_CENTRAL_ADDRESS=_<address>:<port_number>_ <1>
+----
++
+<1> For `<rox_password>`, specify the {rh-rhacs-first} administrator password.
+
+.Procedure
+* Run the following command to generate a diagnostic bundle:
++
+[source,terminal]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" -p "$ROX_PASSWORD" central debug download-diagnostics
+----
```

Comment:
@dhaus67 Please do a technical review of only this portion of the content, every other content you see below this comment is commented out (meaning it won't appear in the docs portal), thanks!

Reply:
Please suggest any changes over here.

Reply:
So, if I understand correctly, only the `ROX_PASSWORD` way will currently be shown. I feel like we should show both (so, on one side using `ROX_API_TOKEN` environment variable, as well as the `ROX_PASSWORD / -p` way.

Otherwise, the contents look good to me.

Reply:
Done.

Reply:
@dhaus67 Please view the changes here: https://github.com/openshift/openshift-docs/pull/56709/commits/9f0985d2dbe28ffcf142411c77ba4bd0ffb70b17